.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,40 @@
+env: {}
+
+# FILE GENERATED WITH: npx ghat fregante/ghatemplates/npm-publish
+# SOURCE: https://github.com/fregante/ghatemplates
+
+# Collaborators can publish a new version of what's on master via "workflow_dispatch"
+# https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
+
+name: Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      Version:
+        description: "Version accepted by `npm version *`"
+        required: true
+
+jobs:
+  NPM:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+          registry-url: https://registry.npmjs.org
+      - run: npm ci || npm install
+      - uses: fregante/setup-git-user@v2
+      - name: Create version
+        # Get the generated version, this enables support for keywords: `npm version patch`
+        run: |
+          VERSION="$(npm version "${{ github.event.inputs.Version }}")"
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@twschiller we need a NPM_TOKEN with write access to webext-messenger. Visit:

https://www.npmjs.com/settings/tschiller/tokens

Reply:
It looks like it was set but expired. It should be good to go now

@grahamlangford links to notes on how to create the token: https://www.notion.so/pixiebrix/Creating-Updating-NPM-Publishing-Tokens-b8f083afbf634ffc923337c8bb44091d?pvs=4

Reply:
For him to be able to do the renewal, he also needs to be listed as a owner in the package, but I only see you on npm: https://www.npmjs.com/package/webext-messenger